src/app/modules/dashboard/modules/day-plan/day-plan.reducer.ts

```diff
@@ -97,7 +100,17 @@ export const selectSelectedDayPlanSummary = createSelector(
   (selectedDay: Day | undefined) => selectedDay ? selectedDay.summary : undefined
 );
 
+export const selectSelectedDayPlanEatenMealsSummary = createSelector(
+  selectSelectedDayPlan,
+  (selectedDay: Day | undefined) => selectedDay ? selectedDay.eatenMealsSummary : undefined
```

Comment:
jest nowa sztuczka na to `  (selectedDay: Day | undefined) => selectedDay && selectedDay.eatenMealsSummary`

Reply:
zrobione